CMakeLists.txt

```diff
@@ -85,6 +85,10 @@ endif()
 include(cmake/utils.cmake)
 find_package(Maya 2018 REQUIRED)
 
+if(APPLE AND BUILD_UB2 AND NOT MAYA_MACOSX_BUILT_WITH_UB2)
+    message(WARNING  "Maya was NOT built with Universal Binary 2")
+endif()
```

Comment:
Simply issue a warning plugin is being built with UB2 but Maya was not. Should not cause any problems, but its extra compilation not needed.